web/src/frontend/src/skjema/styles/variables.less

```diff
@@ -1,3 +1,4 @@
 @import "./node_modules/nav-frontend-core/less/core";
 
 @skjemaMarg: 4rem;
+@skjemabkg: #efefef;
```

Comment:
fargen bør være en variabel fra nav-frontend. Strengt regime for hvilke farger vi får lov å bruke :)

Reply:
Jeg hørte med Erlend, og denne ligger foreløpig ikke inne i nav-frontend-core. Erlend skulle sjekke opp videre om dette var en glipp, og at den skal med i core.

Reply:
Ja, men den er for mørk. Erlend skal sjekke hvorfor den lyse fargen ikke er i nav-frontend-core